packages/pwa/app/utils/routes-utils.test.js

```diff
@@ -7,40 +7,228 @@
+                '/en-uk/category/:categoryId',
+                '/category/:categoryId',
+                '/fr/category/:categoryId',
+                '/fr-FR/category/:categoryId',
+                '/it/category/:categoryId',
+                '/it-IT/category/:categoryId'
+            ]
+        },
+        {
+            urlConfig: {
+                site: 'path',
+                locale: 'query_param',
+                showDefaults: true
+            },
+            expectedRes: [
+                '/uk/',
+                '/site-1/',
+                '/us/',
+                '/site-2/',
+                '/',
+                '/uk/category/:categoryId',
+                '/site-1/category/:categoryId',
+                '/us/category/:categoryId',
+                '/site-2/category/:categoryId',
+                '/category/:categoryId'
+            ]
+        },
+        {
+            urlConfig: {
+                site: 'path',
+                locale: 'query_param',
+                showDefaults: false
+            },
+            expectedRes: [
+                '/',
+                '/us/',
+                '/site-2/',
+                '/category/:categoryId',
+                '/us/category/:categoryId',
+                '/site-2/category/:categoryId'
+            ]
+        },
+        {
+            urlConfig: {
+                site: 'query_param',
+                locale: 'query_param',
+                showDefaults: true
+            },
+            expectedRes: ['/', '/category/:categoryId']
+        },
+        {
+            urlConfig: {
+                site: 'query_param',
+                locale: 'query_param',
+                showDefaults: false
+            },
+            expectedRes: ['/', '/category/:categoryId']
+        },
+        {
+            urlConfig: {
+                site: 'path',
+                locale: 'path',
+                showDefaults: true
+            },
+            expectedRes: [
+                '/',
+                '/uk/en-uk/category/:categoryId',
+                '/uk/en-GB/category/:categoryId',
+                '/uk/fr/category/:categoryId',
+                '/uk/fr-FR/category/:categoryId',
+                '/uk/it/category/:categoryId',
+                '/uk/it-IT/category/:categoryId',
+                '/site-1/en-uk/category/:categoryId',
+                '/site-1/en-GB/category/:categoryId',
+                '/site-1/fr/category/:categoryId',
+                '/site-1/fr-FR/category/:categoryId',
+                '/site-1/it/category/:categoryId',
+                '/site-1/it-IT/category/:categoryId',
+                '/us/en-US/category/:categoryId',
+                '/site-2/en-US/category/:categoryId',
+                '/category/:categoryId'
+            ],
+            ignoredRoutes: ['/']
+        }
+    ]
+    const env = process.env
+    beforeEach(() => {
+        jest.resetModules()
+        process.env = {...env}
+    })
+
+    afterEach(() => {
+        process.env = env
+    })
```

Comment:
`env` or `process.env` does not seem to be used anywhere in this test file.

Reply:
Good catch, I will remove it